src/Widgets/SearchResultsWidget/SearchResultsWidgetComponent.tsx

```diff
@@ -113,9 +142,18 @@ provideComponent(SearchResultsWidget, ({ widget }) => {
           ) : null}
         </div>
       </section>
-    </InPlaceEditingOff>
-  )
-})
+    )
+  },
+  {
+    loading: () => (
+      <section className="bg-white py-3">
+        <div className="container text-center">
```

Comment:
* [ ] FYI The code duplicated in the component and the loading could be moved to the parent component.
* [ ] FYI If we drop the `text-center` the loading becomes trivial. Maybe good enough?

Reply:
I would like to keep the `text-center`. Otherwise the loader looks a bit lost.